VikingQuiz.UI/src/Components/StartGame/StartGame.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import './StartGame.css';
+import HomeButton from '../Buttons/HomeButton/HomeButton';
+import axios from '../../../node_modules/axios';
+import UserMinimalProfile from '../UserMinimalProfile/UserMinimalProfile';
+import CancelButton from '../Buttons/CancelButton/CancelButton';
+import StartButton from '../Buttons/StartButton/StartButton';
+
+class StartGame extends React.Component<any, any> {
+    constructor(props: any) {
+      super(props);
+
+        this.state = {
+            serverMessage: '',
+            redirect: false,
+            player: [],
+            currentPlayerNumber: 0
+        }
+    }
+
+    public componentWillMount() {
+        axios.get('http:///localhost:60151/api/player')
+        .then(response => {
+            console.log(response.data)
+            this.setState({player: response.data})
+        })
+        .catch(err => console.log(err))
+    }
+
+    public getFirstListOfPlayers() {
+        const firstListOfPlayers = [];
+        for( let i = 0; i < 7; i++){
+            firstListOfPlayers[i] = this.state.player[i];
+        }
+        return firstListOfPlayers;
+    }
+
+    public getSecondListOfPlayers() {
+        const secondListOfPlayers = [];
+        for( let i = 7; i < this.state.player.length; i++){
+            secondListOfPlayers[i-7] = this.state.player[i];
+        }
+        return secondListOfPlayers;
+    }
+
+    
+    public render() {
+        const displayedMessage = "YOUR CODE";
+        const displayedCode = "code";
+        let firstListOfPlayers = [];
+        let secondListOfPlayers = [];
+        if (this.state.player.length > 7){
+            firstListOfPlayers = this.getFirstListOfPlayers();
```

Comment:
This is wrong. You should rethink the algorithm so that you don`t need to use an if anymore.